inference-engine/src/vpu/graph_transformer/src/stages/roi_align.cpp

```diff
@@ -22,6 +22,7 @@ static const char s_pooled_w[] = "pooled_w";
 static const char s_pooled_h[] = "pooled_h";
 static const char s_sampling_ratio[] = "sampling_ratio";
 static const char s_spatial_scale[] = "spatial_scale";
+static const char s_use_buffer[] = "use_buffer";
```

Comment:
A redundant string is still here.
But I recommend not to fix it right now to avoid restarting validation. 
I'll remove it with the next PR with firmware updating

Reply:
ok, thanks!